portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,9 +26,31 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
+    private List<String> comments;
+
+    @Override
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add("Yeehaw");
+    comments.add("I am going to live forever or die trying!");
+    comments.add("What is the average air-speed of an unladen swallow?");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Alex!</h1>");
+      String json = convertToJsonUsingGson(comments);
+    response.setContentType("application/json");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
+   */
+  private String convertToJsonUsingGson(List<String> messages) {
+    Gson gson = new Gson();
```

Comment:
Defice gson as a class variable so it does not need to be re-init every-time convertToJsonUsingGson gets called